carta/session.py

```diff
@@ -418,6 +418,50 @@ def clear_raster_scaling_reference(self):
         """Clear the raster scaling reference."""
         self.call_action("clearRasterScalingReference")
 
+    # VIEWER MODES
+    @validate(OneOf(["single", "multiple"]))
```

Comment:
I prefer not to use "magic" string literals like this. Instead, I suggest that you create a constant called `PanelMode` with attributes `SINGLE` and `MULTIPLE`. Since these string values aren't actually used as strings, it should have the default integer values (like e.g. `SmoothingMode`).

Reply:
In 1684398, `PanelMode` is added to `constants.py`:
![截圖 2023-04-12 下午8 42 40](https://user-images.githubusercontent.com/106953609/231460703-d0249d64-d952-4405-aae5-374c70dab3fb.png)
and is then used in `set_viewer_mode`:
![截圖 2023-04-12 下午8 45 16](https://user-images.githubusercontent.com/106953609/231461339-83736f59-8bea-4522-971f-478844b7d84f.png)